app/repository/interfaces/RaceGroupRepository.java

```diff
@@ -1,19 +1,18 @@
 package repository.interfaces;
 
+import com.fasterxml.jackson.databind.JsonNode;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.fasterxml.jackson.databind.JsonNode'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)